components/transfer/transfer.lua

```diff
@@ -0,0 +1,407 @@
+	end
+
+	if icon and iconModule then
+		local iconTemp = iconModule[ string.lower(icon) ]
+		if iconTemp then
+			row = row .. iconTemp .. ' '
+		else
+			mw.log( 'No entry found in Module:PositionIcon/data: ' .. icon .. ' (' .. name .. ')')
+			row = row .. '[[File:Logo filler event.png|16px|link=]][[Category:Pages with transfer errors]] '
+		end
+	end
+
+	row = row .. '[[' .. (link or name) .. '|' .. name .. ']]'
+
+	return row
+end
+
+function Transfer._createTeam(team, teamsec, role, rolesec, isOldTeam, date)
+	local teamCell = mw.html.create('div')
+	teamCell:attr('class', 'divCell Team ' .. (isOldTeam and 'OldTeam' or 'NewTeam'))
+
+	if team == nil and role == nil then
+		teamCell:css('font-style', 'italic')
+		teamCell:wikitext('None')
+		return teamCell
+	end
+
+	if team then
+		teamCell:node(mw.ext.TeamTemplate.teamicon(team, date))
+	end
+	if teamsec then
+		teamCell:node('/' .. mw.ext.TeamTemplate.teamicon(teamsec, date))
+	end
+	teamCell:node(Transfer._createRole(role, rolesec, team))
+
+	return teamCell
+end
+
+function Transfer._createRole(role, rolesec, hasTeam)
+	if role == nil then
+		return nil
+	end
+
+	local span = mw.html.create('span')
+	if hasTeam then
+		span:wikitext('<br/>')
+		span:css('font-size', '85%')
+		span:css('font-style', 'italic')
+		span:wikitext('(' .. role .. (rolesec and '/' .. rolesec or '') .. ')')
+	else
+		span:css('font-style', 'italic')
+		span:wikitext(role .. (rolesec and '/' .. rolesec or ''))
+	end
+
+	return span
+end
+
+function Transfer._createIcon(icon)
+	local div = mw.html.create('div'):attr('class', 'divCell Icon'):css('font-size','larger')
+
+	if icon == nil then
+		div:wikitext('&#x21d2;')
+	else
+		div:wikitext(icon)
+	end
+
+	return div
+end
+
+function Transfer._createReferences(reference, refTable)
+	local div = mw.html.create('div'):attr('class', 'divCell Ref')
+
+	if not(refTable['reference1']) then
+		div:wikitext(reference)
+	else
+		local refPrint = ''
+		for _, fullRef in Table.iter.pairsByPrefix(refTable, 'reference') do
+			local refTemp = mw.text.split(fullRef, ',,,', true)
+			if (refTemp[1] or '') == 'web source' and (refTemp[2] or '') ~= '' then
+				refPrint = refPrint .. '[' .. refTemp[2] .. '<i class="fad fa-external-link-alt wiki-color-dark"></i>]' .. '<br>'
+			elseif (refTemp[1] or '') == 'tournament source' then
+				refPrint = refPrint .. '[[' .. refTemp[2] .. '|' .. 
+					mw.getCurrentFrame():callParserFunction{ name = '#tag:abbr', args = {
+						'<i class="fad fa-link wiki-color-dark"></i>',
+						title = 'Transfer wasn\'t formally announced, but individual represented team starting with this tournament'
+					} } .. ']]<br>'
+			elseif (refTemp[1] or '') == 'inside source' then
+				refPrint = refPrint ..
+					mw.getCurrentFrame():callParserFunction{ name = '#tag:abbr', args = {
+						'<i class="fad fa-user-secret wiki-color-dark"></i>',
+						title = 'Liquipedia has gained this information from a trusted inside source'
+					} } .. '<br>'
```

Comment:
```suggestion
					Abbreviation(
						'<i class="fad fa-link wiki-color-dark"></i>',
						'Transfer wasn\'t formally announced, but individual represented team starting with this tournament'
					) .. ']]<br>'
			elseif (refTemp[1] or '') == 'inside source' then
				refPrint = refPrint ..
					Abbreviation(
						'<i class="fad fa-user-secret wiki-color-dark"></i>',
						'Liquipedia has gained this information from a trusted inside source'
					) .. '<br>'
```
could use `Module:Abbreviation` here instead of calling the parser function